packages/babel-plugin-emotion/src/index.js

```diff
@@ -22,6 +31,30 @@ let primitivesStyledMacro = createStyledMacro({
   isWeb: false
 })
 
+let transformersSource = {
+  emotion: vanillaTransformers,
+  '@emotion/css': {
+    default: coreCssTransformer
+  },
+  '@emotion/core': {
+    // jsx isn't included here because it's handled differently
```

Comment:
what does it mean that it's handled differently? would be good to point some point of reference or an explanation here